.babelrc

```diff
@@ -1,19 +1,39 @@
 {
   "sourceMaps": true,
   "presets": [
-    "es2015",
-    "stage-2"
+    "@babel/preset-env",
+    "@babel/preset-flow"
   ],
   "plugins": [
     ["add-module-exports"],
-    ["transform-es2015-modules-umd"],
-    ["transform-react-jsx", { "pragma": "h" }],
-    ["transform-runtime", {
-      "helpers": false,
-      "polyfill": true,
-      "regenerator": true,
-      "moduleName": "babel-runtime"
-    }],
-    [ "transform-flow-strip-types" ]
+    ["@babel/plugin-transform-modules-umd"],
+    [
+      "@babel/plugin-transform-react-jsx",
+      {
+        "pragma": "h"
+      }
+    ],
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "corejs": 2
```

Comment:
Is this just the same as `helpers: false, polyfill: true, regenerator: true`?

Reply:
nope, `helpers` is `true` now. Not sure why it was `false` to begin with.
`polyfill` doesn't exist as an option anymore, which is good and also not sure why it was `true` to begin with. `regenator` is `true` as default, so it stays the same.